client-app/src/pages/account/components/invoices-filter/index.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div v-if="invoicesFilter && availableInvoicesStatuses">
+    <div class="d-flex flex-wrap flex-sm-row flex-column">
+      <div class="col col-sm-6 col-md-5 col-lg-4">
+        <label for="begin-date">{{ $t("account.invoices.date-filter.from") }}</label>
+        <b-form-datepicker
+          id="begin-date"
+          :value="invoicesFilter.startDate"
+          value-as-date
+          reset-button
+          :label-reset-button="$t('account.invoices.date-filter.reset')"
+          :date-format-options="{ year: 'numeric', month: 'long', day: 'numeric' }"
```

Comment:
Seems like if you will just remove format optons, then everything will start working as expected:
https://bootstrap-vue.js.org/docs/components/form-datepicker#internationalization

Reply:
default date format is different
![image](https://user-images.githubusercontent.com/22875828/77051722-70857880-69d4-11ea-9417-5687582e00b1.png)

Reply:
Default date format depends on your local date format

Reply:
It depends, but default format for both EN and DE is too long